posenet_engine_test.py

```diff
@@ -0,0 +1,72 @@
+# Lint as: python3
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from pose_engine import PoseEngine, KeypointType
+from PIL import Image
+from PIL import ImageDraw
+import test_utils
+import numpy as np
+import unittest
+import sys
+import os
+
+test_image = os.path.join(os.getcwd(), 'test_data/test_couple.jpg')
+
+
+class PoseEngineAccuracyTest(unittest.TestCase):
+
+    def test_runinference_all_models(self):
```

Comment:
Does this test do anything? It's not asserting on the fake output. I guess you're just seeing that inference works and making sure it doesn't hit a error in that call? Doesn't the below test do that too?

Reply:
it's just an inference test, and true, I'll take it out